### PR TITLE
Translation fr advanced path operation advanced configuration

### PR DESCRIPTION
**Pull request by axel584**
Hi

This is a PR to translate docs/advanced/path-operation-advanced-configuration.md to french fr

Thanks !

### CODE REVIEW

File: docs/fr/docs/advanced/path-operation-advanced-configuration.md [hunk 1]
```diff
@@ -1,99 +1,99 @@
...
+Si vous souhaitez utiliser les noms de fonction de vos API comme `operationId`, vous pouvez les parcourir tous et remplacer chaque `operation_id` de *path operation* en utilisant leur `APIRoute.name`.
 
-You should do it after adding all your *path operations*.
+Vous devriez le faire après avoir ajouté toutes vos *paramètres de chemin*.
 
 ```Python hl_lines="2  12-21  24"
 {!../../../docs_src/path_operation_advanced_configuration/tutorial002.py!}
 ```
 
 !!! tip
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/path-operation-advanced-configuration.md:26]: translated in oters documents :) 

```suggestion
!!! Astuce

File: docs/fr/docs/advanced/path-operation-advanced-configuration.md [hunk 2]
```diff
@@ -1,99 +1,99 @@
...
-    If you manually call `app.openapi()`, you should update the `operationId`s before that.
+    Si vous appelez manuellement `app.openapi()`, vous devez mettre à jour les `operationId` avant.
 
 !!! warning
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/path-operation-advanced-configuration.md:29]:
```suggestion
!!! Attention

File: docs/fr/docs/advanced/path-operation-advanced-configuration.md [hunk 3]
```diff
@@ -1,99 +1,99 @@
...
+## Description avancée de docstring
 
-You can limit the lines used from the docstring of a *path operation function* for OpenAPI.
+Vous pouvez limiter le texte utilisé de la docstring d'une *fonction de chemin* qui sera affiché sur OpenAPI.
 
-Adding an `\f` (an escaped "form feed" character) causes **FastAPI** to truncate the output used for OpenAPI at this point.
+L'ajout d'un `\f` (un caractère d'échappement "form feed") va permettre à **FastAPI** de tronquer la sortie utilisée pour OpenAPI à ce stade.
 
-It won't show up in the documentation, but other tools (such as Sphinx) will be able to use the rest.
+Il n'apparaîtra pas dans la documentation, mais d'autres outils (tels que Sphinx) pourront utiliser le reste.
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/path-operation-advanced-configuration.md:48]: wdyt ?
```suggestion
Il n'apparaîtra pas dans la documentation, mais d'autres outils (tel que Sphinx) pourront utiliser le reste.

File: docs/fr/docs/advanced/path-operation-advanced-configuration.md [hunk 4]
```diff
@@ -1,99 +1,99 @@
...
 
 ```Python hl_lines="19-29"
 {!../../../docs_src/path_operation_advanced_configuration/tutorial004.py!}
 ```
 
-## Additional Responses
+## Réponses supplémentaires
 
-You probably have seen how to declare the `response_model` and `status_code` for a *path operation*.
+Vous avez probablement vu comment déclarer le `response_model` et le `status_code` pour une *opération de chemin*.
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/path-operation-advanced-configuration.md:56]: Small remark, starting from this line the `*path operation*`  is translated, but not before
I feel like we could align them, in a direction on in the other :v:

File: docs/fr/docs/advanced/path-operation-advanced-configuration.md [hunk 5]
```diff
@@ -1,99 +1,99 @@
...
 
-That defines the metadata about the main response of a *path operation*.
+Cela définit les métadonnées sur la réponse principale d'une * opération de chemin *.
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/path-operation-advanced-configuration.md:58]: be carful, it is not working with spaces between  
```suggestion
Cela définit les métadonnées sur la réponse principale d'une *opération de chemin*.

File: docs/fr/docs/advanced/path-operation-advanced-configuration.md [hunk 6]
```diff
@@ -1,99 +1,99 @@
...
+Vous pouvez également déclarer des réponses supplémentaires avec leurs modèles, codes de statut, etc.
 
-There's a whole chapter here in the documentation about it, you can read it at [Additional Responses in OpenAPI](./additional-responses.md){.internal-link target=_blank}.
+Il y a un chapitre entier ici dans la documentation à ce sujet, vous pouvez le lire sur [Réponses supplémentaires dans OpenAPI](./additional-responses.md){.internal-link target=_blank}.
 
-## OpenAPI Extra
+## OpenAPI supplémentaire
 
-When you declare a *path operation* in your application, **FastAPI** automatically generates the relevant metadata about that *path operation* to be included in the OpenAPI schema.
+Lorsque vous déclarez un *chemin* dans votre application, **FastAPI** génère automatiquement les métadonnées concernant ce * chemin* à inclure dans le schéma OpenAPI.
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/path-operation-advanced-configuration.md:66]:
```suggestion
Lorsque vous déclarez un *chemin* dans votre application, **FastAPI** génère automatiquement les métadonnées concernant ce *chemin* à inclure dans le schéma OpenAPI.

File: docs/fr/docs/advanced/path-operation-advanced-configuration.md [hunk 7]
```diff
@@ -1,99 +1,99 @@
...
-It has all the information about the *path operation* and is used to generate the automatic documentation.
+Il contient toutes les informations sur le *chemin* et est utilisé pour générer automatiquement la documentation.
 
-It includes the `tags`, `parameters`, `requestBody`, `responses`, etc.
+Il inclut les `tags`, `parameters`, `requestBody`, `responses`, etc.
 
-This *path operation*-specific OpenAPI schema is normally generated automatically by **FastAPI**, but you can also extend it.
+Ce schéma OpenAPI spécifique aux *operations* est normalement généré automatiquement par **FastAPI**, mais vous pouvez également l'étendre.
 
 !!! tip
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/path-operation-advanced-configuration.md:77]:
```suggestion
!!! Astuce

File: docs/fr/docs/advanced/path-operation-advanced-configuration.md [hunk 8]
```diff
@@ -1,99 +1,99 @@
...
-    This is a low level extension point.
+    Il s'agit d'extension de bas niveau.
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/path-operation-advanced-configuration.md:78]: hummmm I think this line is confusing, I would i be in favor of :fire: it ! 
Wdyt ?

File: docs/fr/docs/advanced/path-operation-advanced-configuration.md [hunk 9]
```diff
@@ -124,47 +124,47 @@ And if you see the resulting OpenAPI (at `/openapi.json` in your API), you will
...
+Ensuite, nous utilisons directement la requête et extrayons son contenu en tant qu'octets. Cela signifie que FastAPI n'essaiera même pas d'analyser le payload de la requête en tant que JSON.
 
-And then in our code, we parse that YAML content directly, and then we are again using the same Pydantic model to validate the YAML content:
+Et nous analysons directement ce contenu YAML, puis nous utilisons à nouveau le même modèle Pydantic pour valider le contenu YAML :
 
 ```Python hl_lines="26-33"
 {!../../../docs_src/path_operation_advanced_configuration/tutorial007.py!}
 ```
 
 !!! tip
```
Comment (JulianMaurin) [inline docs/fr/docs/advanced/path-operation-advanced-configuration.md:167]:
```suggestion
!!! Astuce

**Review by JulianMaurin — APPROVED** [review]
Awesome!

Thank you for the quality of your translation and the notable effort you are making to make it more consistent. Thank you also for following the procedure which made the review much easier :heart: 

I have made some comments, except for the one about the italics, I leave them to your judgment :rocket:

**Review by PraveenNanda124 — COMMENTED** [review]
Good work

**Review by rjNemo — APPROVED** [review]

**Review by cfraboulet — APPROVED** [review]
LGTM 👍🏻 Good work 😊

**Pull request closed without merging**